content/blogs/newsletter-104.md

```diff
@@ -0,0 +1,114 @@
+---
+title:       "Newsletter 104 - April/May news"
+author:      "GeoB99"
+date:        2024-04-22
+tags:        [ newsletter ]
+---
+
+Greetings from the ReactOS Team!
+This year, we will start with a short summary highlighting ongoing work on the project as well as announcing the selection of our brand new developers.
+We will also talk about the current situation with symmetric multiprocessing (SMP) support in ReactOS.
+
+## Development team enlargement
+Over the last decade, the number of people in the project team base has not changed much.
+While development continued and still continues as usual, the development team hadn't seen any increase in new developers.
+In fact, some of the existing core developers became inactive for quite a while, some of whom were maintainers of important core modules of the project.
+
+This left the development workflow in a state of paralysis for some of the module components of ReactOS, as nobody else in the team had the right skills to determine the correctness of the patches provided
+by contributors in order to get them shipped in the [master](https://github.com/reactos/reactos) branch.
+The ReactOS Team is currently adopting an approach to circumvent this problem, by enrolling contributors into the development team who have contributed useful patches for at least a year, have good coding skills, have collaborated with project team members and are willing to work in the area in which they are interested.
+
+With that said, the ReactOS Team is pleased to announce the addition of 4 new developers for the project -- Carl J. Bialorucki (cbialorucki), Doug Lyons (Doug-Lyons), Oleg Dubinskiy (oleg-dubinskiy) and Whindmar Saksit (whindsaks).
+Their main focus is to maintain the Shell and Audio components, as well as on any other module that requires assistance.
+
+## Symmetric Multiprocessing (SMP) support status & GCC 13 preparations
+Symmetric multiprocessing support for the AMD64 architecture is being extensively improved thanks to Timo Kreuzer (tkreuzer).
+In particular, he has implemented support for processor freezing and switching ([PR #6581](https://github.com/reactos/reactos/pull/6581)) so that every logical processor is frozen or switched properly.
+In addition, he implemented basic support for IPIs (inter-processor interrupts) which are needed for SMP scheduling, debugging, and more ([PR #6130](https://github.com/reactos/reactos/pull/6130)).
+Where APC and DPC are used the first time their meanings should be included in parenthesis similar to the way IPI was done above.
```

Comment:
This was not intended to be inserted verbatim, but to be a comment only. Suggested change from:
"Where APC and DPC are used the first time their meanings should be included in parenthesis similar to the way IPI was done above."
to:
"These are only limited to APC (Asynchronous Procedure Calls), DPC (Deferred Procedure Calls) and freeze IPIs for the time being."
(comment here, not to be included: Sorry for the confusion. Doug Lyons)